File: taller/AMarotoChivite-taller/AMarotoChivite-taller-Java/src/test/java/simulacionTaller/factories/BossFactoryTest.java
package simulacionTaller.factories;

import org.junit.jupiter.api.Test;
import simulacionTaller.models.Boss;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class BossFactoryTest {

    @Test
    void createTest() {
        Boss boss = BossFactory.create();
        // No nulo, para ver si se han asignado correctamente los datos
        assertNotNull(boss);
    }
}

File: taller/AMarotoChivite-taller/AMarotoChivite-taller-kotlin/src/test/kotlin/simulacionTaller/factories/BossFactoryTest.kt
package simulacionTaller.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class BossFactoryTest {
    @Test
    fun createTest() {
        val boss = BossFactory.create()
        // No nulo, para ver si se han asignado correctamente los datos
        Assertions.assertNotNull(boss)
    }
}